### ISSUE TITLE
Problem to configure multiple instance of mongodb on the same node following Readme instruction

### ISSUE THREAD

**Issue by kikicarbonell** [edelight/chef-mongodb#270]
I have one recipies that it's using chef-mongodb, and i need a configuration to run two instance of mongodb on same node whit diferent config params (ex: port, data path, etc.)

My recipies have only this lines:

```
include_recipe 'mongodb::10gen_repo'
include_recipe 'mongodb::default'

mongodb_instance "mongodb" do
   port 27017
end

mongodb_instance "my_instance" do
   port 27333
   dbpath "/data/"
end
```

And the result was one instance of mongodb that is listening by port 27017 and this process configuration is loading from /etc/mongodb.conf . However i need two process, one that is listening from 27017 port and second from 27333.

I have open question related this problem on [stackoverflow](http://stackoverflow.com/questions/22543896/how-run-multiple-instance-of-mongodb-on-the-same-node-using-mongodb-cookbook)

**Comment by jamesonjlee** [comment]
yeah we currently don't support this.  We are working on turning the definition into a resource so that this would be possible.

**Comment by kikicarbonell** [comment]
jameson, thanks for your quick attention, but i don't understand why cookbook owner publish specific instruction to implement multiple instance on same node in Readme file if isn't available yet.

**Comment by jamesonjlee** [comment]
that is an documentation error afaik.  I don't believe our readme _is_ suppose to say you can run multiple mongo instances in a single node.  Can you point me to the place so it can be updated?

**Comment by kikicarbonell** [comment]
view section Usage: [single-mongodb-instance](https://github.com/edelight/chef-mongodb#single-mongodb-instance) on Readme file (master branch)...

**Comment by jamesonjlee** [comment]
hrm.. that might actually still work if you set the proper variables, but otherwise I believe that to be in error.

**Comment by josephholsten** [comment]
#245 especially mentions the importance of testing multiple mongos instances on the same node.
